workflows/misc_file_types/netCDF_chunk.Rmd

```diff
@@ -0,0 +1,29 @@
+
+## Working with netCDFs
+
+This section is for dealing with netCDF (.nc) files. These files require data tables but since they can not be simply opened on the computer using a text editor or Excel, you can use Panoply to explore them or these R commands:
+
+```{r, eval=FALSE}
+library(arcticdatautils)
+library(ncdf4)
```

Comment:
Instead of calling library calls, can you put the namespace in front of the function?